tasks/collect-gh-params/collect-gh-params.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-gh-params
+  labels:
+    app.kubernetes.io/version: "0.1.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that collects GitHub configuration options from the data file
+  params:
+    - name: dataPath
+      type: string
+      description: Path to the JSON string of the merged data containing the pyxis configuration options to use
+      default: data.json
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      default: mapped_snapshot.json
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  results:
+    - name: repository
+      type: string
+      description: "The github repository where the release should be created"
+    - name: release_version
+      type: string
+      description: "The version string of the new release"
+    - name: githubSecret
+      type: string
+      description: "The kube secret to use to authenticate to GitHub, containing one key: token"
+  steps:
+    - name: collect-gh-params
+      image:
+        quay.io/hacbs-release/release-utils:48a1cd6c939eda25382472135be4afe2c9b4b703
```

Comment:
should be redhat-appstudio image

Reply:
Changed here and in every task modified in this PR